content/docs/1.5.4/high-availability/auto-remount-volume.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Auto Remount Volume By Deleting Workload Pod
+weight: 1
+---
+
+Under following conditions, Longhorn automatically apptempts to deletes the workload pod if the pod is managed by a controller (e.g. deployment, statefulset, daemonset, etc...). By deleting the pod, its controller restarts the pod and Kubernetes handles volume reattachment and remount.
+
+1. Unexpected detachment happens, which can happen during a [Kubernetes upgrade](https://github.com/longhorn/longhorn/issues/703), a [Docker reboot](https://github.com/longhorn/longhorn/issues/686), a network disconnection, or a volume engine crashing.
+2. Automatic Salvage. Volumes will be automatically salvaged when all the replicas become faulty e.g. due to network disconnection. Longhorn will try to figure out which replicas are usable, then use them for the volume.
+3. Share manager goes into error state when using RWX volume.
```

Comment:
```suggestion
When the following situations occur, Longhorn automatically attempts to delete workload pods that are managed by a controller (for example, Deployment, StatefulSet, or DaemonSet). After deletion, the controller restarts the workload pod and Kubernetes handles volume reattachment and remounting.

1. A volume was unexpectedly detached, possibly because of a [Kubernetes upgrade](https://github.com/longhorn/longhorn/issues/703), [Docker reboot](https://github.com/longhorn/longhorn/issues/686), network connectivity issue, or volume engine crash.

1. A volume was automatically salvaged after all replicas became faulty, possibly because of a network connectivity issue. Longhorn will identify the usable replicas and use them for the volume.

1. An error occurs on a Share Manager pod that uses an RWX volume.
```

Reply:
Fixed, thanks